setup.py

```diff
@@ -60,6 +60,7 @@
             "requests >= 2.1.0",
             "requests-oauthlib >= 0.4",
             "future >= 0.9.0",
+            "python-dateutil",
```

Comment:
I don't think we should be adding hard requirements for blog importing, which wont be needed most of the time. It could be an extra like `extras_require={'blogimport': ['python-dateutil']},`